app/assets/stylesheets/sufia/_collections.scss

```diff
@@ -79,3 +79,7 @@ form.per_page {
     padding-left: 0;
   }
 }
+
+.collection_form_visibility .form-group {
+  padding-left: 25px;
```

Comment:
I think we apply this to the upload work form to fix this:
https://github.com/projecthydra/sufia/blob/0f0a4ae509194ac9eb7bbd8006fa526678b2b7f8/app/assets/stylesheets/sufia/_form-progress.scss#L43

Maybe we can use the same style with two selectors?